### DISCUSSION TITLE
CLI without weld?

### DISCUSSION THREAD

**Discussion by sweco-sekrsv** [donmccurdy/glTF-Transform#1009] [Q&A] [answered]
Hi!

I can't figure out how to run the cli optimize command without weld. Is that possible? I see that there are options for simplify and compress but not for weld.
Ideally can I use optimize with a chain of commands like: compress textures with webp and then also compress meshes with draco?

**Comment by donmccurdy** [comment, marked as answer]
Hi @sweco-sekrsv! I think that's a good idea – the CLI doesn't currently include a flag to disable welding in the `optimize` step though, no. I've filed a feature request ticket for it here:

- https://github.com/donmccurdy/glTF-Transform/issues/1014

Is that something you'd want to make a PR for? 

> Ideally can I use optimize with a chain of commands like: compress textures with webp and then also compress meshes with draco?

Otherwise yes, the `optimize` command is just a convenient collection of the library's features. If you're only looking for specific optimizations, or need full access to an optimization's settings (e.g. to configure draco compression) then you could run the CLI commands separately, or run the pipeline as a Node.js script.

Example:

```bash
gltf-transform webp input.glb output.glb
gltf-transform draco output.glb output.glb --quantization-volume scene
```

**Comment by sweco-sekrsv** [comment]
It's out of my league to make a PR for this :)
However I tried what you did suggest and that work just fine. A bit of unnecessary IO but that's ok for my use case.
Thanks!

**Reply by donmccurdy** [reply]
No worries! adding the option in https://github.com/donmccurdy/glTF-Transform/pull/1019 :)
